.github/workflows/main.yml

```diff
@@ -23,20 +20,25 @@ jobs:
         - python-version: "3.12"
           env:
             TOXENV: py
-
-        # PyPY
-        - python-version: pypy3.8
```

Comment:
Any reason not to keep testing PyPy 3.9?

Reply:
PyPy 3.9 is EOL. They have different support cycles.